blog/2023-05-18-how-to-use-conda-build.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: "How to use conda-build to build a python package with C/C++ dependency"
+slug: "2023-05-18-how-to-use-conda-build.md"
+authors: [vasvi]
+tags: [conda]
+image: img/blog/2023-02-01-matrix/conda-community-space-matrix.png
+description: "We are pleased to announce that the conda community has migrated our chat rooms to the Matrix/Element platform."
+---
+import Image from '@theme/IdealImage';
+import condabuild from '@site/static/img/blog/2023-05-18-how-to-use-conda-build.md/conda-build.png';
+
+<Image img={condabuild} />
+
+<hr />
+Conda-build is a conda package that automates the process of building and distributing python packages. It is a powerful tool that automates the process of building and distributing Python packages with the added advantage of handling dependencies that require C/C++ or other languages. This is particularly useful for scientific computing, where many Python packages have complex dependencies and require specialized libraries and tools.
+
+Conda-build creates a "recipe" that outlines the steps needed to build a package from source code. This recipe includes all the information from downloading and installing dependencies to compiling the source code and creating the final package. A recipe typically includes:
+
+<ol>
+<li>A script that contains metadata.</li>
+<li>The script that installs the files for the package on macOS and Linux.</li>
+<li>The build script that installs the files for the package on Windows.</li>
+<li>Any additional setup files depending on the complexity of the package.</li>
+</ol>
+
+In this article, we will guide you through the process of building your python package with conda-build! Conda-build can help us handle even the most complex dependencies for our package, making the process smoother and more efficient. Let's get ready to build an amazing Python package!
+
+## Getting Started 
+
+### Prerequisites
+Let's start building our own Python package. First, you need to ensure that you have installed <a href="https://docs.anaconda.com/anaconda/install/">Miniconda or Anaconda</a>. Conda-build is a package manager that can be installed using either of the two package managers.
```

Comment:
I'd recommend linking to the docs.conda.io page on installing Miniconda separately from the link to Anaconda's install docs.

Reply:
Ok sure